shell/components/LocaleSelector.vue

```diff
@@ -69,7 +69,7 @@ export default {
 </script>
 
 <template>
-  <div>
+  <div role="group">
```

Comment:
hmmm.. it appears that the original axe warning has to do with fallthrough attrs being applied to the root div element of LocaleSelector:

https://github.com/rancher/dashboard/blob/94665787504e732b06827e261b5782031a69db4e/shell/pages/prefs.vue#L200-L203

I think that the proper fix would be to ensure that `aria-labeledby` is applied to the `select` element of LocaleSelector.

Reply:
Fixed